src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,20 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        boolean firstTeamWin = false;
```

Comment:
try to use ternary operator and simplify this solution :)
for example: https://www.baeldung.com/java-ternary-operator